ceph_installer/controllers/osd.py

```diff
@@ -58,9 +58,12 @@ def configure_post(self):
         # even with configuring we need to tell ceph-ansible
         # if we're working with upstream ceph or red hat ceph storage
         extra_vars = util.get_install_extra_vars(request.json)
+        extra_vars['raw_multi_journal'] = True
+        extra_vars['raw_journal_devices'] = request.json["journal_devices"]
         extra_vars.update(request.json)
         del extra_vars['host']
         del extra_vars['monitors']
+        del extra_vars['journal_devices']
         if 'redhat_storage' in request.json:
             del extra_vars['redhat_storage']
```

Comment:
I think all this block of extra_vars handling should be done in a helper method that would take the json as an argument.

That way this controller method becomes leaner and the extra vars are easier to test (no need for a request, just call the method directly)